README.md

```diff
@@ -2,111 +2,157 @@
+The `bpolys` parameter can be passed as a `geopandas.GeoDataFrame` containing the polygon features.
 
+``` python
+bpolys = gpd.read_file("./data/polygons.geojson")
+client.elements.count.groupByBoundary.post(bpolys=bpolys, filter="amenity=restaurant")
+```
 
-##### Boundary
-
-The [boundary](https://docs.ohsome.org/ohsome-api/stable/boundaries.html) of the query can be defined using the `bpolys`, `bboxes` and `bcircles` parameters. The `bpolys` parameter can be passed as a `geopandas.GeoDataFrame`, while `bboxes` and `bcircles` can be provided as ...
+##### bboxes
 
-a list containing the coordinates of the bounding box or circle
+The `bboxes` parameter contains the coordinates of one or several bounding boxes.
 
 ``` python
-bboxes = [8.7137,49.4096,8.717,49.4119]
-bcircles = [8.7137,49.4096, 100]
+bboxes = [8.7137,49.4096,8.717,49.4119] # one bounding box
+bboxes = [[8.7137,49.4096,8.717,49.4119], [8.7137,49.4096,8.717,49.4119]]
+bboxes = {"A": [8.67066, 49.41423, 8.68177, 49.4204],
+	  "B": [8.67066, 49.41423, 8.68177, 49.4204]}
 ```
 
-a list containing several bounding boxes or circles
+##### bcircles
+
+The `bcircles` parameter contains one or several circles defined through the coordinates of the centroids and the radius in meters.
 
 ```python
-bboxes = [[8.7137,49.4096,8.717,49.4119], [8.7137,49.4096,8.717,49.4119]]
+bcircles = [8.7137,49.4096, 100]
 bcircles = [[8.7137,49.4096, 100], [8.7137,49.4096, 300]]
+bcircles = {"Circle1": [8.695, 49.41, 200],
+	    "Circle2": [8.696, 49.41, 200]}
 ```
 
-or a dictionary containing several bounding boxes or circles including user defined IDs.
+#### Time
+
+The [time](https://docs.ohsome.org/ohsome-api/stable/time.html) parameter must be ISO-8601 conform can be passed in several ways
+
+```python
+time = '2018-01-01/2018-03-01/P1M'
+time = ['2018-01-01', '2018-02-01', '2018-03-01']
+time = datetime.datetime(year=2018, month=3, day=1)
+time = pandas.date_range("2018-01-01", periods=3, freq="M")
+```
+
+## Contribution Guidelines
+
+If you want to contribute to this project, please fork the repository or create a new branch containing your changes.
+
+**Install the pre-commit hooks** in our local git repo before commiting to ensure homogenous code style.
 
-``` python
-bboxes = {"A": [8.67066, 49.41423, 8.68177, 49.4204], "B": [8.67066, 49.41423, 8.68177, 49.4204]}
-bcircles = {"C1": [8.695, 49.41, 200], "C2": [8.696, 49.41, 200]}
 ```
+$ pre-commit install
+```
+
+**Run the tests** inside the repo using `pytest` (and `poetry` if you like) to make sure everything works.
 
-##### Time
+```
+$ poetry run pytest
+```
+
+Running the tests in a docker container containing an ohsome API instance is faster, but not mandatory. To set up and start such a docker container run the following command before running the tests.
+
+```
+$ docker run -t -i --name ohsome-api -p 8080:8080 julianpsotta/ohsome-api:latest
```

Comment:
docker run -dt --name ohsome-api -p 8080:8080 julianpsotta/ohsome-api:1.3.2

Reply:
Better to use the detached mode here with the latest version instead of latest. Latest will be the repository latest. Mostly nightly builds etc.

Reply:
ok, changed it.